jupyterhub/notebook-images/overlays/additional/scipy-notebook-imagestream.yaml

```
@@ -3,16 +3,20 @@ kind: ImageStream
 metadata:
   labels:
     opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/thoth-station/s2i-minimal-notebook"
+    opendatahub.io/notebook-image-name: "SciPy Notebook Image"
+    opendatahub.io/notebook-image-desc: "Jupyter notebook image containing basic dependencies for data science and machine learning work."
   name: s2i-scipy-notebook
 spec:
   lookupPolicy:
     local: true
   tags:
   - annotations:
-      openshift.io/imported-from: quay.io/odh-jupyterhub/s2i-scipy-notebook
+      openshift.io/imported-from: quay.io/thoth-station/s2i-scipy-notebook
     from:
       kind: DockerImage
-      name: quay.io/odh-jupyterhub/s2i-scipy-notebook:3.6
-    name: "3.6"
+      name: quay.io/thoth-station/s2i-scipy-notebook:latest
+    name: "latest" #FIXME
```

```suggestion
      name: quay.io/thoth-station/s2i-scipy-notebook:v0.0.1
    name: "v0.0.1"
```
update the image with tag, however latest also point to same tag. 
so this can be disregarded if we would be following the latest.